compiler/luci/lang/include/luci/IR/Nodes/CircleWhere.h

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright (c) 2020 Samsung Electronics Co., Ltd. All Rights Reserved
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef __LUCI_IR_CIRCLE_WHERE_H__
+#define __LUCI_IR_CIRCLE_WHERE_H__
+
+#include "luci/IR/CircleNodeDecl.h"
+#include "luci/IR/CircleOpcode.h"
+
+#include "luci/IR/VariadicArityNode.h"
+
+#include <cassert>
+
+namespace luci
+{
+
+/**
+ * @brief WHERE in Circle
+ */
+class CircleWhere final : public VariadicArityNode<CircleNodeImpl<CircleOpcode::WHERE>>
+{
+public:
+  CircleWhere(bool provide_xy)
+      : VariadicArityNode<CircleNodeImpl<CircleOpcode::WHERE>>(provide_xy ? 3 : 1)
+  {
+    // NOTHING TO DO
+  }
+
+public:
+  loco::Node *cond() const { return at(0)->node(); }
+  void cond(loco::Node *node) { at(0)->node(node); }
+
+  loco::Node *x() const
+  {
+    if (arity() == 3)
+      return at(1)->node();
+    return nullptr;
+  }
```

Comment:
@seanshpark 
What do you think about this design?
According TF documentation where operator can be executed in two modes: with 1 input (condition) and 3 inputs (condition, x, y). I tried to express it explicitly.
Old design was generic: any number of inputs can be set in where operator.

If you think old design is better, we can revert changes.

Reply:
I think it's OK, I'd like to ask @parjong also :)

Reply:
if this returns `nullptr` caller needs to check if it's null or not. this `if` conditional branch can happen with `arity() == ` condition. so condition check has not much difference.
so, this may be about how methods are user friendly. current is +1 :)

Reply:
> According TF documentation where operator can be executed in two modes

@binarman Could you please check TFLITE documentation or implementation? TF and TFLITE are similar in most cases, but we need to check this again.

Reply:
I checked what is generated by converter for TF Where and found that it generates TFLite Where for 1 input mode and TFLite Select for 3 input mode.
I also checked TFLite implementation and found that it is not able to run 3-input Where operator.

Sooo, I think I need to remove 3 input version of Where operator from this PR =)

Reply:
@binarman , FYI, @parjong is on vacation today...